src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -86,11 +87,13 @@
   private final String appCommandOutputFormat;
   private final WaitingProcessOutputLineListener runDevAppServerWaitListener;
 
-  private CloudSdk(Path sdkPath, String appCommandMetricsEnvironment,
-      String appCommandMetricsEnvironmentVersion, @Nullable File appCommandCredentialFile,
-      String appCommandOutputFormat, ProcessRunner processRunner,
-      WaitingProcessOutputLineListener runDevAppServerWaitListener) {
+  private CloudSdk(Path sdkPath, Path javaBinaryPath, String appCommandMetricsEnvironment,
```

Comment:
javaBinaryPath doesn't feel like the usual terminology for this. The common name for this directory is javaHome or jdkHome